tests/e2e/concurrent_query_modules/client.py

```diff
@@ -0,0 +1,33 @@
+import multiprocessing
+
+import mgclient
+import pytest
+
+
+def inner(query, number_of_executions):
+    connection = mgclient.connect(host="127.0.0.1", port=7687)
+    connection.autocommit = True
+    cursor = connection.cursor()
+    for _ in range(number_of_executions):
+        cursor.execute(query)
+    cursor.fetchall()
```

Comment:
you also have GQLAlchemy available in the venv so you can use it as well

Reply:
I didn't know that, but I would prefer to keep it this way if that's fine by you, I am not sure if there is some loss of control with the added abstraction of GQLAlchemy, if we want to extend these tests in the future. Let me know what you think!